app/views/prompts/new.html.erb

```diff
@@ -4,7 +4,7 @@
          <div class="main-slider">
            <div class="col-sm-4 col-sm-offset-4 text-center">
              <h1 class="margin-bottom">New prompt</h1>
-             <%= render 'form' %> 
+             <%# render 'form' %> 
```

Comment:
We will need this form to render elsewhere in the app, yeah? :grimacing: 

At least currently, it is still being linked to throughout many locations (respond to an existing prompt, concept, or character).